frontend/src/components/Organizations.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+ import { ref, onMounted } from 'vue';
+ import Organization from './Organization.vue'
+ import Papa from 'papaparse';
+ const OrgURI = 'https://gist.githubusercontent.com/isaac-looker/bc60ba5a2fe7eeea52be6265ba4b62a6/raw/8fd08e1ee8d1a0f42ddbbfdac09e0133829ad8a1/ex.csv'
```

Comment:
Looks great. Let's try the transition to Google Sheets in a relatively soon iteration in case there are any snags.